frontend/staff-dashboard/src/pages/discounts/show.tsx

```diff
@@ -16,26 +14,49 @@ export const DiscountShow = () => {
     return (
         <Show isLoading={isLoading}>
             <h2>
-                <Space>Discount Detail
-                { record?.for_flexible_pricing ? (<Tag color="green">Flexible Pricing Tier</Tag>) : null }</Space>
+                <Space>
+                    Discount Detail
+                    { record?.for_flexible_pricing ? (<Tag color="green">Financial Assistance Tier</Tag>) : null }
+                    { record?.redemption_type !== "unlimited" && record?.is_redeemed ? (<Tag color="red">Redeemed</Tag>) : null }
+                </Space>
             </h2>
             
-            <Title level={5}>Code</Title>
-            <Text>{record?.discount_code}</Text>
+            <Row>
+                <Col span={12}>
+                    <Title level={5}>Code</Title>
+                    <Text>{record?.discount_code}</Text>
+
+                    <Title level={5}>Redemption Type</Title>
+                    <Text>
+                        <Tag>{record?.redemption_type}</Tag>
+                    </Text>
+
+                    <Title level={5}>Amount</Title>
+                    <Text>{record?.amount} {record?.discount_type}</Text>
+                </Col>
+                <Col span={12}>
+                    {record?.redemption_type === 'unlimited' && record?.max_redemptions > 0 ? (<>
+                        <Title level={5}>Max Redemptions</Title>
+                        <Text>
+                            {record.max_redemptions}
+                        </Text>
+                    </>) : null}
+
+                    <Title level={5}>Activation Date</Title>
+                    <Text>{record?.activation_date ? dayjs(record.activation_date).format("YYYY-MM-DD HH:mm:ss") : (<em>Always Available</em>)}</Text>
 
-            <Title level={5}>Redemption Type</Title>
-            <Text>
-                <Tag>{record?.redemption_type}</Tag>
-            </Text>
+                    <Title level={5}>Expiration Date</Title>
+                    <Text>{record?.expiration_date ? dayjs(record.expiration_date).format("YYYY-MM-DD HH:mm:ss") : (<em>Always Available</em>)}</Text>
+                </Col>
+            </Row>
 
-            <Title level={5}>Amount</Title>
-            <Text>{record?.amount} {record?.discount_type}</Text>
+            {record ? <RedemptionList record={record} key={`redemptions-${record?.id}`} /> : null}
```

Comment:
It would be helpful to update `RedemptionList` to include the user email and some order details.